src/components/elements/ErrorMessage.js

```diff
@@ -1,11 +1,20 @@
 import React from 'react'
 import ErrorIcon from 'material-ui/svg-icons/alert/error'
+import { Card, CardHeader } from 'material-ui/Card'
 import { RetryButton } from 'components/elements/Buttons'
 
-const ErrorMessage = (message, actionForRetry) => <div>
-  <ErrorIcon />
-  <span>{message}</span>
-  { actionForRetry ? <RetryButton actionForRetry={actionForRetry} /> : null }
+const ErrorMessage = (message, actionForRetry, time = null, backgroundColor = null) => <div>
+  <Card
+    className='incident-card'
+    style={{ backgroundColor }}
+  >
+    <ErrorIcon />
+    <CardHeader
+      title={message}
+      subtitle={time ? time.local().format('LTS') : 'Time unknown!'}
```

Comment:
Not sure whether it's preferable to include 'Time unknown!' or to not display a subtitle when time is unknown. I don't have any strong feelings one way or the other, but it may end up being clutter we don't need on the UI.

Another option is to treat null differently from undefined; when the calling code passes in a time value, if it has value then we display the time, if it's undefined we display 'Time unknown!' (because presumably the calling code was trying to pass in a time value), and if it's null (the default case), then the calling code isn't trying to use the time subtitle, and we don't display it. That may be the best of both worlds but increase complexity. Up to you which approach to use (or to stick with this one).

Reply:
Let's leave it with a null subtitle for now when time is undefined, null, or not a moment instance.